src/probabilistic_model/probabilistic_circuit/nx/probabilistic_circuit.py

```diff
@@ -1045,6 +1119,13 @@ def unweighted_edges(self):
 
         return unweighted_edges
 
+
+    def plot(self, **kwargs):
```

Comment:
these should not have plotting anymore

Reply:
i assumed that plotly_layout is not part of it